tests/org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/contentassist/Bug365368Test.java

```diff
@@ -19,7 +19,7 @@
   @Test
   public void testEscapedIdentifierNoPrefix() throws Exception {
     ContentAssistProcessorTestBuilder _newBuilder = this.newBuilder();
-    ContentAssistProcessorTestBuilder _append = _newBuilder.append("class C {\n\t\t\tdef void ^as() {}\n\t\t\tdef void aa() {}\n\t\t\tdef void bb() {}\n\t\t\tdef void m() { this.");
+    ContentAssistProcessorTestBuilder _append = _newBuilder.append("class C {\r\n\t\t\tdef void ^as() {}\r\n\t\t\tdef void aa() {}\r\n\t\t\tdef void bb() {}\r\n\t\t\tdef void m() { this.");
```

Comment:
I'm pretty sure the PreferenceStoreWhitespaceInformationProvider is used along the the LineSeparatorHarmonizer. All our projects should have the preference for new text file line delimiter set to unix. If you see unrelated changes of that kind, that's clearly a bug that needs to be addressed.